### PR TITLE
Food Expansion 4! The great salad update and more

### PR DESCRIPTION
**Pull request by MidoriWroth**
## About The Pull Request

This is my fourth food expansion PR, adding and changing quite a few things.
This PR will:

- Add 7 different types of salads
- Add a new ingredient supply beacon box called 'Salads', which includes ingredients used for the salad recipes
- Give cherry jelly its own container, and allow jelly to be ordered from the produce console
- Make cherry jelly actually nourish you when consumed
- Add paçoca, a Brazilian peanut candy requested by @GuillaumePrata 
- Add mashed potatoes
- Add shepherds pie
- Add Cullen skink

image

## Why It's Good For The Game

Not only is there a severe lack of salad-type foods, but the majority of the currently craftable salads are either gimmicks or get you high from space drugs when eaten, which is a bit silly. This PR will add a big variety of actual popular salads for the vegetarian and health-conscious station dwellers.

I plan on adding more specific food ingredient containers in the future, so I thought I'd start with cherry jelly. Cherry jelly is not hard to come by, but I see no reason for it to not be ordered since it is now getting more use from my previous PRs.

And of course, more foods add more variety and creativity to chef menus

## Changelog

:cl:
add: Added 7 new types of various salads. Vegetarians rejoice!
add: Added a new ingredient order beacon box, 'Salads'. It contains many ingredients for the new salads.
add: Added mashed potatoes
add: Added shepherds pie, because station dwellers gotta grow big and strong somehow
add: Added Cullen skink
add: Added paçoca, a Brazilian peanut dessert
add: Cherry jelly has its own container, which can be made at a condi-master
add: Cherry jelly can be ordered at a produce console
add: Consuming jelly now actually nourishes you
imageadd: Added sprites for all the new tasty foods
/:cl:

### CODE REVIEW

**Comment by BlueMemesauce** [comment]
I don't think honey should be orderable as it is quite a powerful chem

**Comment by Jacquerel** [comment]
I also think that you should need to do beekeeping to get honey, but I like the rest of the salads.

**Comment by GuillaumePrata** [comment]
> I don't think honey should be orderable as it is quite a powerful chem

The real issue IMO is that honey is a stupidly good general medical chem where it should have stayed as cooking/flavorful chem instead.
I'm not an expert but a quick google fu told me that honey is good for healing wounds and burns, would be a nice ghetto way to deal with wounds instead of our version now which is a boring weaker Omnizine.

**Comment by Jacquerel** [comment]
Even if honey wasn't a powerful healing chemical I think it is good for it to only be produced by beekeeping, we don't need the chef to be able to make every single kind of food available without interacting with other departments.
If that means they get made less often that's fine.

**Comment by MidoriWroth** [comment]
Very well, I’ll make it unorderable, but I’m keeping the bottle in for condimaster purposes =)

**Comment by MrDoomBringer** [comment]
Could you make the honey and jelly bottle a bit smaller (e.g. closer to salt/pepper shaker size)? A condiment bottle's sprite taking up an entire tabletop always looks a bit weird to me

**Comment by MidoriWroth** [comment]
> Could you make the honey and jelly bottle a bit smaller (e.g. closer to salt/pepper shaker size)? A condiment bottle's sprite taking up an entire tabletop always looks a bit weird to me

image
How's that?

**Comment by optimumtact** [comment]
just remove the healing from honey it's stupid

**Comment by MrDoomBringer** [comment]
@MidoriWroth Looks good 👍

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm [hunk 1]
```diff
@@ -275,7 +275,28 @@
 	name = "Onigiri"
 	reqs = list(
 		/obj/item/food/salad/boiledrice = 1,
-		/obj/item/food/seaweedsheet = 1,
+		/obj/item/food/seaweedsheet = 1
```
Comment (dragomagol) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm:278]:
```suggestion
		/obj/item/food/seaweedsheet = 1,
```
This list should still end with a comma 
image

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm [hunk 2]
```diff
@@ -275,7 +275,28 @@
...
 	result = /obj/item/food/onigiri
 	subcategory = CAT_MISCFOOD
+
+/datum/crafting_recipe/food/mashed_potatoes
+	name = "Mashed potatoes"
+	reqs = list(
+		/obj/item/food/grown/potato = 2,
+		/obj/item/food/grown/garlic = 1,
+		/obj/item/food/butter = 1,
+		/datum/reagent/consumable/milk = 1
```
Comment (dragomagol) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm:289]:
```suggestion
		/datum/reagent/consumable/milk = 1,
```

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm [hunk 3]
```diff
@@ -275,7 +275,28 @@
...
+	)
+	result = /obj/item/food/mashed_potatoes
+	subcategory = CAT_MISCFOOD
+
+/datum/crafting_recipe/food/pacoca
+	name = "Pacoca"
+	reqs = list(
+		/obj/item/food/grown/peanut = 2,
+		/datum/reagent/consumable/sugar = 5,
+		/datum/reagent/consumable/salt = 2
```
Comment (dragomagol) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm:299]:
```suggestion
		/datum/reagent/consumable/salt = 2,
```

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_pie.dm [hunk 4]
```diff
@@ -216,3 +216,16 @@
...
+
+/datum/crafting_recipe/food/shepherds_pie
+	name = "Shepherds pie"
+	reqs = list(
+		/obj/item/food/mashed_potatoes = 1,
+		/obj/item/food/meat/cutlet = 3,
+		/obj/item/food/grown/onion = 1,
+		/obj/item/food/grown/peas = 1,
+		/obj/item/food/grown/corn = 1,
+		/obj/item/food/grown/garlic = 1
```
Comment (dragomagol) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_pie.dm:228]:
```suggestion
		/obj/item/food/grown/garlic = 1,
```

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_salad.dm [hunk 5]
```diff
@@ -108,3 +108,89 @@
...
 	subcategory = CAT_SALAD
+
+/datum/crafting_recipe/food/kale_salad
+	name = "Kale salad"
+	reqs = list(
+		/obj/item/reagent_containers/cup/bowl = 1,
+		/obj/item/food/grown/carrot = 1,
+		/obj/item/food/onion_slice/red = 2,
+		/obj/item/food/grown/cabbage = 1,
+		/datum/reagent/consumable/quality_oil = 2
```
Comment (dragomagol) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_salad.dm:119]:
```suggestion
		/datum/reagent/consumable/quality_oil = 2,
```

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_salad.dm [hunk 6]
```diff
@@ -108,3 +108,89 @@
...
+	subcategory = CAT_SALAD
+
+/datum/crafting_recipe/food/greek_salad
+	name = "Greek salad"
+	reqs = list(
+		/obj/item/reagent_containers/cup/bowl = 1,
+		/obj/item/food/grown/tomato = 1,
+		/obj/item/food/onion_slice/red = 2,
+		/obj/item/food/cheese/wedge = 1,
+		/datum/reagent/consumable/quality_oil = 5
```
Comment (dragomagol) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_salad.dm:131]: You get the idea, these should all have trailing commas

**Review by dragomagol — CHANGES_REQUESTED** [review]
Do we need the honey png if we're not adding it to the order console?

**Comment by MidoriWroth** [comment]
Ah okay, every other food has the last ingredient not with a comma, so I just followed that pattern.

**Comment by VladinXXV** [comment]
I like your greek salad, but why no olives? The sprite is missing them and so is the recipe. #67239 added olives, so I think you should be able to add them in there.

**Comment by MidoriWroth** [comment]
> I like your greek salad, but why no olives? The sprite is missing them and so is the recipe. #67239 added olives, so I think you should be able to add them in there.

oops lmao, i had it in my pre-planned recipe but i never included it in the code, thanks!

File: code/game/objects/items/food/soup.dm [hunk 7]
```diff
@@ -307,3 +307,10 @@
 	tastes = list("creamy vegetables"= 2, "sausage" = 1)
 	foodtypes = VEGETABLES | MEAT | GRAIN | BREAKFAST
 
+/obj/item/food/soup/cullen_skink
+	name = "cullen skink"
+	desc = "A thick Scottish soup made of smoked fish, potatoes and onions.."
```
Comment (dragomagol) [inline code/game/objects/items/food/soup.dm:312]:
```suggestion
	desc = "A thick Scottish soup made of smoked fish, potatoes and onions."
```

Comment (dragomagol) [reply code/game/objects/items/food/soup.dm:312]: Other than this one thing, lgtm

**Review by dragomagol — APPROVED** [review]

**Review by Imaginos16 — APPROVED** [review]
Its fine.

**Pull request merged**